themes/default/content/blog/pulumi-and-localstack/index.md

```diff
@@ -64,7 +64,7 @@ Despite the documentation, it can be quite confusing and becomes even more so wh
 
 The big thing to remember is that a Pulumi "output" value [is not known until after the infrastructure has been fully provisioned](https://www.pulumi.com/docs/concepts/inputs-outputs/). They are a promise of a future value. If you want to refer to the value in other infrastructure code, Pulumi creates a dependency graph to be able to resolve the value at the right time. If, however, you want to refer to the value at runtime and you want to define that runtime code in a modular way, you have to be very clear about how that value is presented by Pulumi to that runtime code.
 
-Our infrastructure code provisions a DynamoDB table and an AWS Lambda. That Lambda needs to know the name of that table at runtime in order to query it. Because the *actual* name of the table isn't known until provisioning is underway (Pulumi creates many infrastructure resources based on the logical name but with suffixes to avoid naming collisions), we need a way to get the "output" table name from Pulumi's `aws.dynamodb.Table` [API](https://www.pulumi.com/docs/reference/pkg/aws/dynamodb/table/#example-usage) to the Lambda we'll create with the "simpler" way to create a Lambda using Pulumi's `cloud.API` [API](https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/cloud/#API).
+Our infrastructure code provisions a DynamoDB table and an AWS Lambda. That Lambda needs to know the name of that table at runtime in order to query it. Because the *actual* name of the table isn't known until provisioning is underway (Pulumi creates many infrastructure resources based on the logical name but with suffixes to avoid naming collisions), we need a way to get the "output" table name from Pulumi's `aws.dynamodb.Table` [API](https://www.pulumi.com/registry/aws/dynamodb/table/#example-usage) to the Lambda we'll create with the "simpler" way to create a Lambda using Pulumi's `cloud.API` [API](https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/cloud/#API).
```

Comment:
```suggestion
Our infrastructure code provisions a DynamoDB table and an AWS Lambda. That Lambda needs to know the name of that table at runtime in order to query it. Because the *actual* name of the table isn't known until provisioning is underway (Pulumi creates many infrastructure resources based on the logical name but with suffixes to avoid naming collisions), we need a way to get the "output" table name from Pulumi's `aws.dynamodb.Table` [API](https://www.pulumi.com/registry/packages/aws/dynamodb/table/#example-usage) to the Lambda we'll create with the "simpler" way to create a Lambda using Pulumi's `cloud.API` [API](https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/cloud/#API).
```